javascript/javascript3/week1/API+Explanation.js

```diff
@@ -0,0 +1,78 @@
+// my json file about football DataTransferItemList
+
+
+{
+    "teams": [
+      {
+        "name": "Real Madrid",
+        "country": "Spain",
+        "league": "La Liga",
+        "stadium": "Santiago Bernabeu",
+        "founded": 1902
+      },
+      {
+        "name": "Manchester United",
+        "country": "England",
+        "league": "Premier League",
+        "stadium": "Old Trafford",
+        "founded": 1878
+      },
+      {
```

Comment:
I would put this in an actual .json file and import it if needed. Now it is useless.